models/collection.js

```diff
@@ -64,7 +71,8 @@ module.exports = {
         'id',
         'name',
         'pipelineIds',
-        'pipelines'
+        'pipelines',
+        'type'
```

Comment:
let's not make the `type` field required
just make it optional so we don't need to do a DB migration